opencti-platform/opencti-graphql/src/resolvers/user.js

```diff
@@ -68,6 +69,7 @@ const userResolvers = {
     roles: (_, args, context) => findRoles(context, context.user, args),
     creators: (_, args, context) => findAll(context, context.user, args),
     assignees: (_, args, context) => findAll(context, context.user, args),
+    members: (_, args, context) => findAllMembers(context, context.user, args),
```

Comment:
I don't find where it's used in front-end

Reply:
in ObjectMembersField.tsx